### ISSUE TITLE
Cannot find native ssl

### ISSUE THREAD

**Issue by arn-the-long-beard** [sfackler/rust-openssl#1512]
Hello guys :smile:  ! 

Well, I ran into issue with `openssl-sys` and it is not the first time but i fixed the variables. But now I have this.

Here are my variables 
```
export OPENSSL_DIR="/etc"
export OPENSSL_INCLUDE_DIR="/usr/include/openssl"
```

And I get this when I want to use a library called `cargo deb`
 
```
   Compiling openssl-sys v0.9.63
   Compiling openssl v0.10.34
error: could not find native static library `ssl`, perhaps an -L flag is missing?

error: aborting due to previous error

error: could not compile `openssl-sys`
```

I already spent much time to set the variables, but now I am kind of lost. Can somebody help me with this  ? 

It would be so nice to understand what is happening there :D

**Comment by sfackler** [comment]
> OPENSSL_DIR - If specified, the directory of an OpenSSL installation. The directory should contain lib and include subdirectories containing the libraries and headers respectively.

https://docs.rs/openssl/0.10.35/openssl/#manual

**Comment by arn-the-long-beard** [comment]
Hey thank you for your message :smile: 

I tried to do what the documentation says for several hours last time before I posted here, but I still get the errors even when I set the right path on my linux arch. I am not sure what I am missing actually. 

> The directory should contain lib and include subdirectories containing the libraries and headers respectively.

I understand the meaning of it. That is why I got this error message. 

Is there a way to know which directory I should target ?
